metaflow/runner/metaflow_runner.py

```diff
@@ -282,6 +283,20 @@ def __get_executing_run(self, tfp_runner_attribute, command_obj):
         )
         return ExecutingRun(self, command_obj, run_object)
 
+    async def __async_get_executing_run(self, attribute_file_fd, command_obj):
+        content = await async_handle_timeout(
+            attribute_file_fd, command_obj, self.file_read_timeout
+        )
+        content = json.loads(content)
+        pathspec = "%s/%s" % (content.get("flow_name"), content.get("run_id"))
+
+        # Set the correct metadata from the runner_attribute file corresponding to this run.
+        metadata_for_flow = content.get("metadata")
+        metadata(metadata_for_flow)
```

Comment:
we are no longer setting the `metadata` explicitly.. i.e `metadata(metadata_for_flow)` needs to be removed..

refer to the implementation of `__get_executing_run` and you will see the following..

```py
# Set the correct metadata from the runner_attribute file corresponding to this run.
metadata_for_flow = content.get("metadata")

run_object = Run(
    pathspec, _namespace_check=False, _current_metadata=metadata_for_flow
)
```

Reply:
Thanks, fixed! Must've missed it while rebasing.